Codecademy/gamut
# docs: fix CourseCard story

### Overview


Updates the storybook examples for the CourseCard to remove imageSrc as a field being passed (since it's not a prop of CourseCard)



### PR Checklist


#### Testing Instructions

[Visit CourseCard story in the storybook preview 
](https://github.com/Codecademy/gamut/pull/2677)
Note that none of the examples provide 'imageUrl' as a prop to the CourseCard component

Before:
(see `imageSrc` being passed as a prop, but that image isn't the one that's being rendered on the card. course card has a hardcoded image)
<img width="988" alt="Screen Shot 2023-02-10 at 2 20 34 PM" src="https://user-images.githubusercontent.com/9753446/218179131-51737607-e0a2-41c3-8b10-cbd955ef7b2e.png">

After:
(no `imageSrc` and an image is still visible :) 
<img width="1015" alt="Screen Shot 2023-02-10 at 2 36 20 PM" src="https://user-images.githubusercontent.com/9753446/218182258-c32de5af-cc8b-4685-8711-94df3de154b5.png">


#### PR Links and Envs

Given this is just a story book update -- I did not create a PRs in mono & monolith to test, but let me know if you would like to me to! 

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
